api/src/test/java/io/kafbat/ui/AbstractIntegrationTest.java

```diff
@@ -32,6 +34,7 @@
 public abstract class AbstractIntegrationTest {
   public static final String LOCAL = "local";
   public static final String SECOND_LOCAL = "secondLocal";
+  protected static final int timeout = 100;
```

Comment:
I belive we could get rid of this variable as it doesn't seem reusable anyway

Reply:
Done